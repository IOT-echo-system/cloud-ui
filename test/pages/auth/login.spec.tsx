import React from 'react'
import {render} from '@testing-library/react'
import LoginPage from '../../../src/pages/auth/login'

jest.mock('../../../src/templates/accounts')
jest.mock('../../../src/templates/accounts/login/Login')

describe('Login page test', () => {
  beforeEach(jest.clearAllMocks)
  afterEach(jest.clearAllMocks)

  it('should match with the snapshot', () => {
    const {container} = render(<LoginPage />)

    expect(container).toMatchSnapshot()
  })
})
