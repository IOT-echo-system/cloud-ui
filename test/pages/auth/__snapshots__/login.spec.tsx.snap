// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login page test should match with the snapshot 1`] = `<div />`;
